data/config_files/example_config_random.json

```diff
@@ -143,6 +143,19 @@
       "focalLength": 50.0
     }
   },
+  "occlusion":{
+    "render-occlusion": true,
```

Comment:
please do not change the behaviour of example_config_random.json
to exploit the new settings generate a file named example_config_random_occlusion.json or similar
this file should keep the behaviour and not render occlusions